Makefile

```diff
@@ -48,6 +48,8 @@ vendor: ## Vendor dependencies
 ## Test
 
 test: ## Run all tests
+	$(GO) install github.com/vektra/mockery/v2@v2.34.2
+	mockery
```

Comment:
That only works if `~/go/bin` is part of the users path. `$(go env GOPATH)/bin/mockery` would be more reliable.

But I'd prefer to not make changes to a users environment. `go run github.com/vektra/mockery/v2@v2.34.2` would also work, but takes half a second longer.

Reply:
fixed.